### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 8

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#334] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324607-e383550d-e83e-4dfe-b3a1-7383bd1b6656.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217805601-17c99f83-1121-445a-a7da-eb6749ca685d.png)

**Comment by LuginiAndrea** [comment]
La terza è vera:
$A \in$ Sat implica che $\exists \alpha$ t.c. $\alpha\left(A\right) = 1$. Inoltre non ci è detto se $B$ è una tautologia, quindi ipotizziamo che sia $\in$ Sat ma $\not\in$ Taut.
Ne segue che $A = 1 \land B = 0$ è una coppia possibile, il che implica che $$B \implies A \land A \centernot\implies B = 0 \implies 1 \land \neg\left(1 \implies 0\right) = 1 \land 1 = 1$$.
Ne segue che un $B$ con queste proprietà esiste
